configure.py

```diff
@@ -1951,11 +1975,18 @@ def query_seastar_flags(pc_file, link_static_cxx=False):
                 header_objs=' '.join(["$builddir/{mode}/{hh}.o".format(mode=mode, hh=hh) for hh in headers])
             )
         )
+        f.write(
+            'build {mode}-wasm: phony {wasms}\n'.format(
+                mode=mode,
+                wasms=' '.join(['$builddir/{}/{}'.format(mode, binary) for binary in sorted(wasms)]),
+            )
```

Comment:
I don't think wasms should be mode-specific, they don't have distinct flags, do they?